usaspending_api/common/helpers/generic_helper.py

```diff
@@ -18,11 +18,11 @@
 )
 from usaspending_api.references.models import Agency
 
-logger = logging.getLogger(__name__)
 
+logger = logging.getLogger(__name__)
 TEMP_SQL_FILES = [str(DEFAULT_MATIVEW_DIR / val["sql_filename"]) for val in MATERIALIZED_VIEWS.values()]
-
 VIEW_SQL_FILES = [str(val) for val in OVERLAY_VIEWS]
+REFRESH_MATVIEWS_SQL = " ".join(f"refresh materialized view {matview_name};" for matview_name in MATERIALIZED_VIEWS)
```

Comment:
The SQL to refresh matviews is stored here: usaspending_api/database_scripts/matviews/refresh_matviews.sql